src/main/java/com/logdyn/model/repositories/JiraRepository.java

```diff
@@ -123,13 +123,13 @@ else if (resCode == 401)
             return Optional.of(new Task(key, title, desc, new URL(this.url, ISSUE_BROWSER_PATH + key)));
 
         } catch (final JSONException e) {
-            LOGGER.debug(String.format("Exception parsing JSON in createTask() of Jira Repository '%s'", this.name), e);
+            LOGGER.error(String.format("Exception parsing JSON in createTask() of Jira Repository '%s'", this.name), e);
```

Comment:
Makes sense

Reply:
Will revert to debug since this is an antipattern anyway - https://community.oracle.com/docs/DOC-983543 Antipatterns "Log and Return Null" (wrt to exception handling an empty Optional is equivalent to null)